includes/Core/Dismissals/REST_Dismissals_Controller.php

```diff
@@ -95,6 +96,39 @@ protected function get_rest_routes() {
 					'permission_callback' => $can_dismiss_item,
 				)
 			),
+			new REST_Route(
+				'core/user/data/dismissed-items',
+				array(
+					'methods'             => WP_REST_Server::DELETABLE,
+					'callback'            => function ( WP_REST_Request $request ) {
+						$slugs = $request['data']['slugs'];
+
+						foreach ( $slugs as $slug ) {
+							$this->dismissed_items->remove( $slug );
+						}
+
+						return new WP_REST_Response( $this->dismissed_items->get_dismissed_items() );
+					},
+					'permission_callback' => $can_dismiss_item,
+					'args'                => array(
+						'data' => array(
+							'type'                 => 'object',
+							'required'             => true,
+							'minProperties'        => 1,
+							'additionalProperties' => false,
+							'properties'           => array(
+								'slugs' => array(
+									'type'     => 'array',
```

Comment:
It's worth noting that if a string is passed for `slugs` it will be coerced to an array, this is the expected behaviour as discussed under [Primitive Types > Type Juggling](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#primitive-types) in the WordPress docs.